pkg/label/label.go

```diff
@@ -32,6 +33,13 @@ func AppVersionSelector(unique bool) labels.Selector {
 	return selector
 }
 
+func ChartOperatorAppSelector(unique bool) string {
+	return fmt.Sprintf("%s=%s,%s=%s", label.AppOperatorVersion,
+		GetProjectVersion(unique),
+		label.AppKubernetesName,
+		key.ChartOperatorAppName)
+}
```

Comment:
This selector gets the correct chart-operator app CR because it may be named chart-operator or chart-operator-unique.